ci-operator/step-registry/rosa-qe/sts/cluster/nightly/rosa-qe-sts-cluster-nightly-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: rosa-qe-sts-cluster-nightly
+  steps:
+    env:
+      ACCOUNT_ROLES_PREFIX: "rosa-sts-nightly"
+      CLUSTER_NAME: "sdq-nrosa-sts"
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
+      MULTI_AZ: "true"
+      ENABLE_AUTOSCALING: "true"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+      CHANNEL_GROUP: "nightly"
+      ETCD_ENCRYPTION: "true"
+      DISABLE_WORKLOAD_MONITORING: "false"
+    pre:
+    - ref: rosa-sts-account-roles-create
```

Comment:
Per common OCP convention,  suggest to create standalone `*-provision` and `*-deprovision` chains for creating and destroying cluster, e.g. `cucushift-installer-rehearse-aws-ipi-ovn-provision` and `cucushift-installer-rehearse-aws-ipi-ovn-deprovision`